migration/sql-files/002-tracker-items.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE ONLY tracker_items
```

Comment:
No, dropping the column will not work with current running version in demo environment without the code change introduced in this PR.